### PR TITLE
for #6 : Visualise evaluation metric

### PR DESCRIPTION
**Pull request by KaairaGupta**
For #6 
The task was to plot the mean and spread of `y` vs `x`.
I have plotted the following vs x with input as defined in the task:
- mean of y 
- maximum and minimum values of y
- spread of y around mean (standard-deviation)

### CODE REVIEW

File: dev/KaairaGupta/evaluation metric/example_visualise_evaluation_metric.ipynb [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = []\n",
+    "\n",
+    "x = []\n",
+    "\n",
+    "# Creating simple data for [x1,x2,x3,...x4]\n",
+    "for i in range(10):\n",
+    "    x.append(i)\n",
```
Comment (dzeber) [inline dev/KaairaGupta/evaluation metric/example_visualise_evaluation_metric.ipynb]: Alternatively, `np.arange(10)`

**Comment by KaairaGupta** [comment]
@dzeber Do you want me to remove the standard deviation band and just show a band between min and max y?
Also, I was thinking to plot multiple graphs, one as above and other with violin plots. Do you think I should add that?

**Comment by mlopatka** [comment]
@KaairaGupta I'll jump in for @dzeber here since he is away until Wednesday.

> @dzeber Do you want me to remove the standard deviation band and just show a band between min and max y?
Yes, you should update the visualization to include only the min/max bounds.

> Also, I was thinking to plot multiple graphs, one as above and other with violin plots. Do you think I should add that?
Yes! I think violin plots would be very informative in complimenting the above-mentioned plot.

**Comment by KaairaGupta** [comment]
@dzeber @mlopatka 
I have made all required changes. PTAL.

**Pull request merged**